client/src/components/redbook/Role.js

```diff
@@ -1,4 +1,9 @@
 import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faGraduationCap } from '@fortawesome/free-solid-svg-icons'
+import { faChalkboardTeacher } from '@fortawesome/free-solid-svg-icons'
+import { faBook } from '@fortawesome/free-solid-svg-icons'
```

Comment:
You can import the different font awesome icons like so

`import { faGraduationCap, faChalkboardTeacher, faBook } from '@fortawesome/free-solid-svg-icons'`

it is cleaner